CONTRIBUTING.adoc

```diff
@@ -0,0 +1,115 @@
+= Contributing to the Asciidoctor Docker Container
+:source-highlighter: coderay
+
+== Requirements
+
+You need the following tools:
+
+* A bash compliant command line
+* link:http://man7.org/linux/man-pages/man1/make.1.html[GNU make]
```

Comment:
nitpick: `link` is optional and I like to extract URL at the top of the file:

```
:gnu-make-url: http://man7.org/linux/man-pages/man1/make.1.html

* {gnu-make-url}[GNU make]
```

Also, I think we should `https` URL

Reply:
Thanks @ggrossetie ! I've added the `https` link.

About the URL on top of the file: what is the rationale behind? I tend to do it when I repeat the URL on the file at least twice, but there might a good practise here that I don't know


Regarding the optional `link`, I tend to it because we have a Markdown file (for use on the DockerHub) so I tend to be explicit to make it easier differentiating between both format.

Reply:
I find it easier to update (potentially broken) links when they are all together at the top of the file. When in doubt, you can <kbd>ctrl</kbd>+<kbd>click</kbd> all of them since they are located at the top of your file and see if they are still working as expected. It's a bit more tedious when external links are scattered.

> Regarding the optional link, I tend to it because we have a Markdown file (for use on the DockerHub) so I tend to be explicit to make it easier differentiating between both format.

👍🏻

Reply:
Make sense! If you don't mind, I'll proceed with this PR (as it is a nitpick) and I'll send a subsequent PR with:

- replacing remaining http:// scheme by https:// (we're in 2024 so either these are dead links to remove or they can handle https:// scheme)
- Moving the links on top of the adoc documentations as described here